x/stakeibc/keeper/hooks.go

```diff
@@ -270,9 +271,15 @@ func (k Keeper) TransferExistingDepositsToHostZones(ctx sdk.Context, epochNumber
 				continue
 			}
 			delegateAddress := delegateAccount.GetAddress()
-			// TODO(TEST-89): Set NewHeight relative to the most recent known gaia height (based on the LC)
 			// TODO(TEST-90): why do we have two gaia LCs?
-			timeoutHeight := clienttypes.NewHeight(0, 1000000)
+			blockHeight, found := k.GetLightClientHeightSafely(ctx, hostZone.ConnectionId)
```

Comment:
Looking through `GetLightClientHeightSafely`: do we need to `return 0, false` in `msg_server_submit_tx.go: GetLightClientHeightSafely:259`?